kafka/src/main/java/com/bazaarvoice/emodb/kafka/KafkaProducerConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package com.bazaarvoice.emodb.kafka;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Optional;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+public class KafkaProducerConfiguration {
+    @Valid
+    @NotNull
+    @JsonProperty("bufferMemory")
+    private Optional<Long> _bufferMemory = Optional.empty();
```

Comment:
Is `Optional.empty` the same as Guava's `Optional.absent`?

Reply:
Yep!